src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -648,8 +648,8 @@ uct_tcp_sockcm_ep_server_invoke_conn_req_cb(uct_tcp_sockcm_ep_t *cep)
     ucs_debug("fd %d: remote_data: (field_mask=%"PRIu64") "
               "dev_addr: %s (length=%zu), conn_priv_data_length=%zu",
               cep->fd, remote_data.field_mask,
-              ucs_sockaddr_str((const struct sockaddr*)remote_data.dev_addr,
-                               peer_str, UCS_SOCKADDR_STRING_LEN),
+              ucs_sockaddr_str(client_saddr.addr, peer_str,
```

Comment:
shall we also update format string (dev_addr -> saddr) and length (remote_data.dev_addr_length -> saddr_len) ?

Reply:
good catch, actually we should print only device address, which doesn't include port.
i've changed the format:
```
[1656582810.859082] [swx-ucx01:53238:a]   tcp_sockcm_ep.c:655  UCX  DEBUG fd 15: remote_data: (field_mask=15) dev_addr: flags 0x0 af 2 in_addr 127.0.0.1 (length=6), conn_priv_data_length=20
```
wdyt?